dao/src/main/java/se/fortnox/reactivewizard/db/ObservableStatementFactory.java

```diff
@@ -111,6 +97,21 @@ private static void closeSilently(Connection connection) {
         return new DaoObservable<>(result, transactionHolder);
     }
 
+    private void scheduleWorker(Subscriber<?> subscription, Action0 action) {
+        Scheduler.Worker worker = scheduler.createWorker();
+        worker.schedule(() -> {
+            try {
+                action.call();
+            } catch (Exception e) {
+                if (!subscription.isUnsubscribed()) {
+                    subscription.onError(e);
+                }
+            } finally {
+                worker.unsubscribe();
```

Comment:
This is the important part?

Reply:
Yes, according to the documentation on Scheduler.createWorker(): "When work is completed it should be unsubscribed using Scheduler.Worker#unsubscribe()."